infrastructure/data/Pulumi.staging.yaml

```diff
@@ -1,4 +1,4 @@
 config:
   aws:region: eu-west-2
```

Comment:
while doing this, noticed that region is only explicitly set on staging - should we set this consistently?